soda/src/main/java/it/okkam/opendata/models/soda/SodaClassification.java

```diff
@@ -9,6 +9,7 @@
   String domain_category;
   String[] domain_tags;
   Map<String, String>[] domain_metadata;
+  Map<String, String>[] domain_private_metadata;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "domain_private_metadata" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00116)